metadata-io/src/test/java/com/linkedin/metadata/AspectUtilsTest.java

```diff
@@ -54,7 +54,7 @@ public void testAdditionalChanges() {
     DatasetProperties datasetProperties = new DatasetProperties().setName("name");
     proposal1.setAspect(GenericRecordUtils.serializeAspect(datasetProperties));
     proposal1.setEntityType("dataset");
-    proposal1.setChangeType(ChangeType.PATCH);
+    proposal1.setChangeType(ChangeType.UPSERT);
```

Comment:
@RyanHolstien - Reminder to discuss this. I do not think that we should be generating additional default aspects because I believe that PATCH should only apply to existing entities. If they exist, then these should have been generated already.

Reply:
going to revert this